alerts.tf

```diff
@@ -73,4 +73,24 @@ module "div-bulkcase-errors-alert" {
   trigger_threshold_operator = "GreaterThan"
   trigger_threshold = 0
   resourcegroup_name = "${azurerm_resource_group.rg.name}"
+}
+
+module "div-data-extraction-alert" {
+  source = "git@github.com:hmcts/cnp-module-metric-alert"
+  location = "${var.location}"
+
+  app_insights_name = "div-${var.env}"
+
+  alert_name = "Data extraction to Family Man has not happened yet"
+  alert_desc = "Logs aren't indicating that e-mails with data extraction were sent in div-${var.env}."
+  app_insights_query = "traces | where message startswith 'Sent extracted data to' and tostring(customDimensions['LoggerName']) has 'dataextraction'"
+  custom_email_subject = "Alert: Data extraction does not seem to be working in div-${var.env}"
+  frequency_in_minutes = 120
+  time_window_in_minutes = 1440
```

Comment:
why 1440? this means you'll keep sending notification every 2 hour for 1 schedule failure

Reply:
ok I think is 1 per day , make sense to set  frequency_in_minutes to 1440 too?

Reply:
My idea here was that in a period of 24 hours, we should always have at least 3 e-mails sent. This check will run every 2 hours (to make sure we know about it in time).
Do you think the config is going to achieve this, @qzhou-hmcts?

Reply:
Your schedule only runs at 2.00am without retry configuration, if it fails it will do at that time and the schedule job won't run again until next day at 2.00 am. that means during 24h you will trigger the same alert (12 in total). Setting 24h frequency with 24h you will cover if during that day the email has been delivered and you will trigger a single alert for that failure

Reply:
That's true. However, we have a manual trigger for that, if it happens.
So the idea is that we get a message every 2 hours until we run it manually.

The problem I thought of with setting one check every 24hrs is that we don't have control over what time that will be triggered. So we might only find out about it too late (i.e 23 hours later)

Reply:
Does that make sense?

Reply:
A quick look doesn't show we can control the time an alert is triggered, only frequency. 
We can try it as it is, or reduce it to run twice/3 times a day (that way it'll run at least once during working hours). And with https://github.com/hmcts/div-case-orchestration-service/pull/532 we're able to trigger it for any given date right?